src/Checkboxes/Checkboxes.js

```diff
@@ -33,13 +33,16 @@ const Checkboxes = ({
           disabled,
           required,
           label,
+          show = true,
           id = uuidv1()
         } = item;
       
+        if (!show) return <Fragment key={id} />;
+
         return (
-          <span
+          <div
                 key={id}
-                className="form-field form-field--checkbox"
+                className={`form-field form-field--checkbox ${required ? 'required' : ''}`}
```

Comment:
Double check with @suzanneb9 because this is not BEM convention compliant.

Reply:
👍

Reply:
Yes, the class is wrong, needs to be BEM 
className="form__field form__field--checkbox"